### ISSUE TITLE
Age Filtering in RSS

### ISSUE THREAD

**Issue by Helipad166** [sabnzbd/sabnzbd#2952]
### Description

I have recently moved from NZBGet to SABnzbd. One thing I cannot figure out is how to filter RSS by age. My use case is I get my NZBs from RSS and on this particular website, the NZBs do not get generated until a few hours later, but the RSS will contain torrent links. In NZBGet, I set the age > 6h, to filter out the really new feeds. But I can't do this in SABnzbd and it ends up with errors in the download page. 

I investigated the RSS link and they can do maxage but not the other way around :( 

Is there a way around this or can age filtering be added as a feature?

**Comment by sanderjo** [comment]
http://127.0.0.1:8080/config/switches/#propagation_delay

Posts will be paused until they are at least this age. Setting job priority to Force will skip the delay.

**Comment by Safihre** [comment]
> http://127.0.0.1:8080/config/switches/#propagation_delay
> 
> Posts will be paused until they are at least this age. Setting job priority to Force will skip the delay.

That's not what he wants I think.

But indeed, we don't support age based filtering indeed. And interestingly, it wasn't requested so far even though it seems like a good feature.

**Comment by Helipad166** [comment]
> http://127.0.0.1:8080/config/switches/#propagation_delay
> 
> Posts will be paused until they are at least this age. Setting job priority to Force will skip the delay.

No, propagation delay does not help in this case as SABnzbd will still try to retrieve NZB from an RSS link that is not available. 

The RSS link is https://feed.animetosho.org/rss2, somehow new feed do not generate NZB until a few hours later but torrent link is active. I think an easy fix (which worked on NZBget) is to put an age limit on this. 

> > http://127.0.0.1:8080/config/switches/#propagation_delay
> > Posts will be paused until they are at least this age. Setting job priority to Force will skip the delay.
> 
> That's not what he wants I think.
> 
> But indeed, we don't support age based filtering indeed. And interestingly, it wasn't requested so far even though it seems like a good feature.

If you can implement this, it would be awesome - conceptually it shouldn't be too hard to add another item in the drop to filter by age field - but then again I am not a programmer.

**Comment by Safihre** [comment]
Re-opening because age filtering is a valid feature request.
